developer-community/tech-talks.md

```diff
@@ -0,0 +1,240 @@
+Examples [here](https://twitter.com/srcd_/status/836158701618802688) and [here](https://twitter.com/srcd_/status/871647537123479553).
+
+• Announcement of Sale start (speakers list might not be complete)
+Examples [here](), [here](https://twitter.com/srcd_/status/873848034484006912) and [here](https://twitter.com/srcd_/status/874173998715875328).
+
+• Sale Start (speakers list might not be complete)
+Examples [here](https://twitter.com/srcd_/status/864036667002425346), [here](https://twitter.com/srcd_/status/864399082487521280) and [here](https://twitter.com/srcd_/status/874189703444090882).
+
+• Optional: If sold out, waiting list ones
+Examples [here](https://twitter.com/srcd_/status/836876422660620288) and [here](https://twitter.com/srcd_/status/841954400876785665).
+
+• Schedule Speakers ones
+Examples [here](https://twitter.com/srcd_/status/839424717886992384), [here](https://twitter.com/srcd_/status/839782571294752768), [here](https://twitter.com/srcd_/status/864156684482215936) and [here](https://twitter.com/srcd_/status/877066994377519104).
+
+• Schedule Printing materials pics ones
+Examples [here](https://twitter.com/srcd_/status/844951764944240641), [here](https://twitter.com/srcd_/status/845321360507318272) and [here](https://twitter.com/srcd_/status/870684058879012864).
+
+• On the day
+Examples [here](https://twitter.com/srcd_/status/845566900851658753), [here](https://twitter.com/srcd_/status/870922426451984384) and [here](https://twitter.com/srcd_/status/878570613404827648).
+
+# Offline
+## Venue
+At the office:
+Talks are held at Margaret Hamilton and chairs should be set for 75 people. The bathroom on that side needs to be sealed and the only two operational are the ones in the kitchen. Kitchen is where we serve the food and all breaks happen - it should be completely empty and cleaned before and tables put against the wall. Both fridges should only contain things that people can drink/eat. At the door, there should be a table blocking access to the two main rooms in the office so that people go straight to the kitchen.
+
+When the event is outside of Madrid, it's done on a case by case basis but we should definitely look for a place that can host at least 75-100 people. Past editions were held:
+• ML: Kl10ch (Contact: Natalia Kosyanova <nk@kl10.ch>)
+
+## Printing Material
+#### Badges/Agenda
+Both Maximo & Esther have the details of the printing company. These need to be sent Thursday latest and they'll deliver them on the next day. Marcelo, or whoever joins as a Designer, will need to get all the specs at the beginning of the week but will know what to send to the printing.
+
+#### Stickers
+We order our stickers, and any other we print, from StickerMule. The more you order (both in terms of quantity of a single design and in terms of numbers of different design) the cheapest it is, of course. So ideally, let Esther know about this in advance so we can pack that order together with any other :)
```

Comment:
Would it be useful to provide the time needed for the stikers in this section? Because the ones of prev section requires only a couple of days...